base/services/tftpd/CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
 
+if(NOT MSVC AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 8.0)
```

Comment:
We don't care about GCC 7 and 6?

Reply:
They don't seem to have that option/warning.